### PR TITLE
#134 FIX

### PR DESCRIPTION
**Pull request by TikhomirovSergey**
Change List:
- #134 FIX. The DeviceTime property was added and it works with Appium
node 1.5

-  improvements of locking methods. The LockDevice(seconds) is obsolete
and it is going to be removed in the next release. Since Appium node
server v1.5.x it is recommended to use
AndroidDriver.Lock()()...AndroidDriver.Unlock() or IOSDriver.Lock(int
seconds) instead.

### CODE REVIEW

**Comment by TikhomirovSergey** [comment]
@Astro03 @imurchie Could you take a look at this PR.

**Comment by TikhomirovSergey** [comment]
#136

**Comment by imurchie** [comment]
This looks reasonable to me, without any knowledge of the language.

File: appium-dotnet-driver/Appium/AppiumDriver.cs [hunk 1]
```diff
@@ -898,6 +901,19 @@ public void Zoom(IWebElement el)
 
         #endregion
 
+        #region Device Time
+
```
Comment (Astro03) [inline appium-dotnet-driver/Appium/AppiumDriver.cs]: add doc tags here.. would be good for people using this class to know what this property is used for.

**Comment by Astro03** [comment]
looks silky otherwise. i like how you put the obsolete tag on there :)

**Pull request merged**